core/arch/arm/plat-imx/platform_config.h

```diff
@@ -128,8 +129,16 @@
 #define GICD_OFFSET			0x1000
 #define GIC_CPU_BASE			(GIC_BASE + GICC_OFFSET)
 #define GIC_DIST_BASE			(GIC_BASE + GICD_OFFSET)
+
+#if defined(PLATFORM_FLAVOR_mx6qsabrelite) || \
+	defined(PLATFORM_FLAVOR_mx6qsabresd)
 #define UART1_BASE			0x02020000
 #define UART2_BASE			0x021E8000
+#else
+#define UART1_BASE			0x02020000
```

Comment:
Actually, I had a look at the datasheet and it appears that:
- UARTs 1, 3 and 5 are at the same addresses on both SoC families.
- UARTs 2 and 4 are only present in the Dual/Quad family.

What do you think about adding all possible UARTs to clarify everything and ease the process to port on any i.MX6 based board ? That would be something like this:

```
#define UART1_BASE          0x02020000                                              
#define UART2_BASE          0x021E8000                                              
#define UART4_BASE          0x021F0000                                              
                                                                                
#if defined(PLATFORM_FLAVOR_mx6qsabrelite) || \                                     
    defined(PLATFORM_FLAVOR_mx6qsabresd)                                            
#define UART3_BASE          0x021EC000                                              
#define UART5_BASE          0x021F4000                                              
#endif 
```